### DISCUSSION TITLE
🚀 Manifest Roadmap Announcement! 🚀

### DISCUSSION THREAD

**Discussion by SebConejo** [mnfst/llm-gateway#132] [Announcements]
Dear Manifest Community,

We are excited to share with you our upcoming release roadmap! Your feedback and support have been invaluable, and we're committed to delivering features that enhance your experience. Here's what's coming:

- [x] **OpenAPI Doc** (July 2024)
  Instant live documentation for your REST API

- [x] **Auth** (Q3 2024)
  Make any entity authenticatable and add rules for endpoints

- [x] **Validation** (Q4 2024)
  Custom validation for creating and updating items

- [x] **Many-to-many** (Q4 2024)
  Advanced relationship management

- [x] **Media upload** (Q4 2024)
  Allow file and image uploads with rules and resizing
- [x] **Hooks** (Q4 2024)
  Trigger webhooks at specific events

Stay tuned by starring our repository, and thank you for being a part of the Manifest journey!

Best, 

The Manifest Team 🦚

**Comment by go4cas** [comment]
@SebConejo ... really cool project! I would like to add an item to roadmap, if possible:

- **Realtime Endpoints/SDK**: Websockets and Server Sent Events(SSE))

**Reply by SebConejo** [reply]
Hi @go4cas,

Thank you! Interesting. We're taking all requests into account for the moment. Tell me, why are you more interested in this feature than any other? What kind of application do you want to make using this feature? What front-end technology do you use? Thanks for your time,

**Reply by SebConejo** [reply]
#nother point @go4cas , could you instead, create a feature request here :https://github.com/mnfst/manifest/discussions/new?category=feature-request

It will be better to follow the request, with more visibility moreover. Other developers will be able to upvote it. 

Thank you,

**Reply by go4cas** [reply]
Thanks, @SebConejo! I have added the Feature Request [here](https://github.com/mnfst/manifest/discussions/212).

**Comment by brunobuddy** [comment]
Closing this out. It's from Manifest's old backend-as-a-service days, which we don't work on anymore since the pivot. See #742 for where things are now.
